ops/dev/vagrant/dev_appserver.sh

```diff
@@ -24,18 +24,34 @@ datastore_args=""
 application=""
 env=""
 
+# Setup Google Application Credentials, if available
+google_application_credentials=$(get_config_prop google_application_credentials)
+if [ $google_application_credentials ]; then
+    cred_file=$(realpath $google_application_credentials)
+    project=$(get_project_from_key "$cred_file")
+    application="--application $project"
+    env="$env --env_var GOOGLE_APPLICATION_CREDENTIALS=$cred_file"
+else
+    application="--application test"
+fi
+
+function assert_google_application_credentials {
+    if [[ "$google_application_credentials" = "null" ]]; then
```

Comment:
cc @phil-lopreiato Is there a better way to do this? I was trying to see if we could have `get_config_prop` return a proper empty string instead of a `null` but I have no idea what I'm doing

Reply:
This seems fine, although if you'd rather make the `get_config_prop` return an empty string if the key is not found, you could add a `| select (.!=null)` in the jq query, making it 

```
jq -r -s ".[0] * (.[1] // {}) | .$prop_name | select (.!=null)" tba_dev_config.json $dev_config_file
```